wearipedia/devices/fitbit/fitbit_sense_fetch.py

```diff
@@ -0,0 +1,100 @@
+import requests
+
+__all__ = ["fetch_real_data"]
+
+
+def call_API(access_token: str, url: str, call: str = "GET"):
+    headers = {"Authorization": "Bearer " + access_token}
+    return requests.request(call, url=url, headers=headers).json()
+
+
+def fetch_real_data(
+    data_type, access_token, start_date="", end_date="", single_date=""
```

Comment:
start_date and end_date should not have a default value, since these seem to be necessary parameters (lemme know if otherwise).